src/app/app.component.ts

```diff
@@ -162,29 +162,36 @@ export class AppComponent implements OnInit {
     }
 
     // The exchange rate requires getting the current Bitcoin price in USD.
-    this.httpClient.get<any>("https://api.blockchain.com/ticker").subscribe(
-      (res: any) => {
-        if (res.USD != null && res.USD.last != null) {
-          this.globalVars.usdPerBitcoinExchangeRate = res.USD.last;
-          // nonaperunit / satoshiperunit / usdperbitcoin * satoshiperbitcoin
-          const nanosPerUnit = 1e9;
-          const satoshisPerBitcoin = 1e8;
-          this.globalVars.nanosPerUSDExchangeRate =
-            (nanosPerUnit /
-              this.globalVars.satoshisPerBitCloutExchangeRate /
-              this.globalVars.usdPerBitcoinExchangeRate) *
-            satoshisPerBitcoin;
-          this.bitcloutToUSDExchangeRateToDisplay = this.globalVars.nanosToUSD(1e9, null);
-          // TODO: When we get rid of the old app, we will only use the globalVars version of this.
-          this.globalVars.bitcloutToUSDExchangeRateToDisplay = this.globalVars.nanosToUSD(1e9, 2);
-
-          this.ref.detectChanges();
+    this.httpClient
+      .get<any>("https://api.blockchain.com/ticker")
+      .subscribe(
+        (res: any) => {
+          if (res.USD != null && res.USD.last != null) {
+            this.globalVars.usdPerBitcoinExchangeRate = res.USD.last;
+          } else {
+            // We divide by 100 to convert from cents to dollars. usdPerBitcoinExchangeRate is expected to be in dollars.
+            this.globalVars.usdPerBitcoinExchangeRate = this.globalVars.ProtocolUSDCentsPerBitcoinExchangeRate / 100;
```

Comment:
Do we know that `ProtocolUSDCentsPerBitcoinExchangeRate` is populated when this call happens? E.g. does this call happen after `GetExchangeRate`?

Reply:
yup -- the only usage of this function is in the subscription to `_updateBitCloutExchangeRate` which calls `GetExchangeRate`